Sources/Overload/OvEditor/include/OvEditor/Core/CameraController.h

```diff
@@ -124,10 +124,10 @@ namespace OvEditor::Core
 		double m_lastMousePosX = 0.0;
 		double m_lastMousePosY = 0.0;
 		OvMaths::FVector3 m_ypr;
-		float m_mouseSensitivity = 0.05f;
-		float m_cameraDragSpeed = 0.01f;
+		float m_mouseSensitivity = 0.12f;
+		float m_cameraDragSpeed = 0.07f; 
 		float m_cameraOrbitSpeed = 0.5f;
-		float m_cameraMoveSpeed = 5.0f;
+		float m_cameraMoveSpeed = 8.0f;
```

Comment:
This setting shouldn't get changed as it is exposed in the UI (Required when working with environments with different scales)